src/cow-react/modules/wallet/api/pure/StatusIcon/index.tsx

```diff
@@ -1,10 +1,10 @@
-import { ConnectionType } from 'connection'
+import { ConnectionType } from '@cow/modules/wallet'
 import styled from 'styled-components/macro'
 
-import CoinbaseWalletIcon from '../../assets/coinbase.svg'
-import FortmaticIcon from '../../assets/formatic.png'
-import WalletConnectIcon from '../../assets/walletConnectIcon.svg'
-import { Identicon } from '../Identicon'
+import CoinbaseWalletIcon from '../../../api/assets/coinbase.svg'
```

Comment:
Should we use absolute paths here?

Reply:
in principle they are in the same module, so i guess relative is also fine. We could do it, i don't have a strong preference

Reply:
Addressed